tests/test_checksums.cpp

```diff
@@ -40,6 +40,7 @@ static const unsigned char raw_tcp_pkt[66] = {
   0x21, 0x9b                                      /* !. */
 };
 
+#if ENABLE_ME_WHEN_USED
```

Comment:
if it is unused, remove this block of code. I will always be able to find it later if I need to.